tools/cli/pkg/ers/instance.go

```diff
@@ -0,0 +1,13 @@
+package ers
+
+// Instance defines a struct for data stored by the ERS
+type Instance struct {
+	Id              string `json:"id"`
```

Comment:
I would propose to use ID instead of Id. See https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
To be fixed in the following PRs